src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+
+    @Override
+    public String getReport(ShopService shopService) {
+        StringBuilder stringBuilder = new StringBuilder("fruit,quantity \n");
+        for (Map.Entry<String, Integer> entry: shopService.getFruits().entrySet()) {
+            stringBuilder.append(entry.getKey())
+                    .append(",")
+                    .append(entry.getValue())
+                    .append("\n");
```

Comment:
use `System.lineSeparator()`

Reply:
.append(System.lineSeparator());

after that error
ERROR]   ReportGeneratorImplTest.testGenerateReport_withFruits:28 expected: <fruit,quantity 
banana,50
apple,100
> but was: <fruit,quantity 
banana,50
apple,100
>

Reply:
already understood the reason